layers/README.md

```diff
@@ -0,0 +1,85 @@
+# layers
+
+This folder contains modules and scripts for working with ATT&CK layers. The core module allows users to load, validate, process, and save ATT&CK layers. A brief overview of the components can be found below. All scripts adhere to the MITRE ATT&CK Navigator Layer file format, [version 3.0](https://github.com/mitre-attack/attack-navigator/blob/develop/layers/LAYERFORMATv3.md).
+
+| script | description |
+|:-------|:------------|
+| [exceptions.py](core/exceptions.py) | Implements errors and supporting functions for the core module. |
```

Comment:
My stance on this is similar to _layerobj - I'd rather document it to help people figure out what's going on behind the scenes should they want to dig into it than leave it out entirely. If you disagree with this, I can absolutely remove this and _layerobj from the documentation.

Reply:
My perspective is that the README should document the API for users who don't want to look at the files. The files should ideally be well documented or intuitive enough to explain things themselves. Especially in the case of exceptions I think the purpose is clear from the name.

Users will probably never import exceptions or LayerObj directly, so documenting it seems like it's just adding additional stuff for the user to read which the average person won't ever need. It's like internal functions. In layerops you have `_grabList` which isn't documented in the API because users don't ever need to call that.